src/locale/locales/it/messages.po

```diff
@@ -4463,11 +4463,11 @@ msgstr "Nessuno"
 #: src/components/LikedByList.tsx:79
 #: src/components/LikesDialog.tsx:99
 msgid "Nobody has liked this yet. Maybe you should be the first!"
-msgstr "Nessuno ha fatto ancora un like. Fai il primo tu!"
+msgstr "Nessuno ha messo like. Fallo tu per prim*!"
```

Comment:
Will the `*` be well-understood by most users? In other strings, the `o/a` is used.